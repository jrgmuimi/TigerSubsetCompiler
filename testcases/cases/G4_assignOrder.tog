let
    var a: int
    var b: int
    var c: int
    var d: int
    var e: int
    var f: int
    var g: int
    var h: int
in
    a := 0;
    b := a + 1;
    c := a;
    d := b+c;
    e := 12;
    f := d-b;
    g := e+f;
    h := g;
    printint(a);printstring("\n");
    printint(b);printstring("\n");
    printint(c);printstring("\n");
    printint(d);printstring("\n");
    printint(e);printstring("\n");
    printint(f);printstring("\n");
    printint(g);printstring("\n");
    printint(h)
end