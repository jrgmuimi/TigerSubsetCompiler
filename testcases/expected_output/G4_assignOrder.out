MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

0
1
0
1
12
0
12
12
